Impossible-Agent-Suite/staxmigrate/src/agent.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  Finding,
+  FindingType,
+  FindingSeverity,
+  TransactionEvent,
+  HandleTransaction
+} from 'forta-agent';
+
+let IF_ADDR = '0xb0e1fc65c1a741b4662b813eb787d369b8614af1';
```

Comment:
variable not used in the agent

Reply:
should be used in the export default

Reply:
`IF_ADDR` is now used in the `export` in `agent.ts`.